features/api_token_settings.feature

```diff
@@ -0,0 +1,16 @@
+
+Feature: Checking create API token
+
+Background:
+    Given we are on Hipchat Login Page
+    When we enter login
+    And we enter password
+    Then we see Welcome title
+
+Scenario: Check create API token
+    Given we are on Account settings Page
+    When we click API access
+    When we reenter password
+    Then we are on API access Page
+    When we create new API token
+    Then we see new API token
```

Comment:
I'd suggest to read why EOF without new line is considered a bad style https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file